web-client/public/gene/info.js

```diff
@@ -19,6 +19,7 @@
     // https://stackoverflow.com/questions/8648892/convert-url-parameters-to-a-javascript-object
 
     var api = window.api;
+    console.log(obj.symbol + typeof obj.symbol);
```

Comment:
Good to lose the earlier `console.log`, but it seems to be compensated for by this one. Was this an oversight? In general we try to restrict console logs to when we debug, removing them before commits (with the one exception being the console.log where we display the resulting network after loading a file—but that's more like the exception that proves the rule).